src/app/collective-rewards/allocations/components/BuilderAllocationHeader.tsx

```diff
@@ -28,13 +37,11 @@ export const BuilderAllocationHeader: FC<BuilderAllocationHeaderProps> = ({
         <Typography tagVariant="label" className="font-semibold line-clamp-1 text-wrap text-base leading-4">
           <AddressOrAlias addressOrAlias={builderName || address} className="text-base font-bold leading-4" />
         </Typography>
-        {gauge && !state.communityApproved && (
-          <Badge content="Deactivated" className="bg-[#932309] color-text-primary py-1 px-1 text-[12px]" />
-        )}
-        {state.paused && state.communityApproved && (
-          <Badge content="Paused" className="bg-[#F9E1FF] text-secondary py-1 px-1 text-[12px]" />
+        {gauge && isBuilderActive(state) ? (
+          <Paragraph className="text-sm font-light"> Joined {date}</Paragraph>
+        ) : (
+          BuilderStateBadges[getBuilderInactiveState(state)]
```

Comment:
nitpick
```
<Badge content={state} className=`${badgeStylePerState[state]} py-1 px-1 text-[12px]` />
```

or even better perhaps to have BuilderStateBadge component. but that's a refactor we might get to at some point.